src/sentry/workflow_engine/models/json_config.py

```diff
@@ -0,0 +1,22 @@
+from abc import abstractproperty
+from typing import Any
+
+from django.db import models
+from jsonschema import ValidationError, validate
+
+
+class JSONConfigBase(models.Model):
```

Comment:
I had added tests for this here: https://github.com/getsentry/sentry/pull/80710/commits/8abfecc8f696c3496615e2ec4d5e8eb5b9e00991 - but removed them cause it was causing a bunch of other failures. any recommendations on how to fix these?